.travis.yml

```diff
@@ -0,0 +1,49 @@
+# We use conda to install cromwell.
+
+language: python
+
+python:
+  - 3.6
+
+services:
+  - docker
+
+before_install:
+  # Install conda
+  - export MINICONDA=${HOME}/miniconda
+  - export PATH=${MINICONDA}/bin:${PATH}
+  - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -f -p ${MINICONDA}
+  - conda config --set always_yes yes
+  - conda config --add channels defaults
+  - conda config --add channels bioconda
+  - conda config --add channels conda-forge
+
+install:
+  - conda install --file requirements-test.txt
+
+script:
+  - pytest --tag $TEST tests/
+
+matrix:
+  include:
+    - env: TEST=single_sample
+    - env: TEST=two_sample
+    # Below is for testing if all files are valid and if all submodules are up to date.
+    - env: TEST=Womtool validate and submodule up to date
```

Comment:
Don't you need to escape the spaces in `TEST` for `TEST=Womtool validate and submodule up to date`

Reply:
This environment variable is not used except as a label for travis. Technically it is not needed.